app/src/main/java/edu/uc/groupProject/topten/DTO/Strawpoll.kt

```diff
@@ -2,6 +2,14 @@ package edu.uc.groupProject.topten.DTO
 
 import com.google.gson.annotations.SerializedName
 
-data class Strawpoll(@SerializedName("id") var id: Int, @SerializedName("title") var title: String,
+/**
+ * Data class that represents a strawpoll
+ *
+ * @param id
+ * @param title
+ * @param votes
+ * @param topTenListID
+ */
+data class Strawpoll(var id: Int, var title: String,
```

Comment:
As the names of fields coming in from the data source are the same as the ones in this DTO class, there is no need to use @SerializedName.